<launch>
  <!-- <node name="controllInterface" pkg="head_pkg" type="controllInterfaceRev2.py" output="screen" launch-prefix="sudo"/> -->
  <node name="speechNode" pkg="head_pkg" type="speechNode.py" output="screen"/>
  <node name="headInterface" pkg="head_pkg" type="HeadInterface.py" output="screen"/>
</launch>
